aptos-move/move-examples/data_structures/sources/smart_table.move

```diff
@@ -0,0 +1,358 @@
+/// A smart table implementation based on linear hashing. (https://en.wikipedia.org/wiki/Linear_hashing)
```

Comment:
Can you add documentation on how to create and use smart_table here? Specifically, I think a developer flow would be great